packages/devtools/devtools/src/panels/client/LoggingPanel/LoggingPanel.tsx

```diff
@@ -83,11 +85,20 @@ export const LoggingPanel = () => {
     setLogs((logs) => [...logs.slice(-MAX_LOGS), logEntry]);
   }, [logEntry]);
 
+  const presets = [
```

Comment:
@yarolegovich @dmaretskyi could you add some useful presets here? One in particular that we'd like to see is edge replication logs

Reply:
separately, we've all discussed getting a feed of events from the edge services that we could surface here.